### ISSUE TITLE
Questions about the words on model improvement in readme

### ISSUE THREAD

**Issue by tqyunwuxin** [hzwer/arXiv2020-RIFE#64]
1、"We recommend generating optical flow labels on 2X size images for better labels. ”

question: Shouldn't the output size of optical flow network be the same as the input image？and if I get optical flow labels on 2X size images ,how to use it？

2、“You can also generate labels during training, or finetune the optical flow network on the training set. "
question: "finetune"  ---do you mean training the RIFE and  the pretrained   optical flow network together?

3、the model v1.5 in [https://github.com/hzwer/arXiv2020-RIFE/issues/41](url) is trained by rife_hd?

**Comment by hzwer** [comment]
1. Many optical flow network provider smaller output than input, if you generate smaller or bigger label, use bilinear resize and rescale value. (such as 2X bilinear resize and *2 all flow value)

2. In my experiments, using census loss or reconstrction loss to finetune optical flow network on Vimeo90K for 5 epoch will slightly improve the final PSNR.

3. Yes, and we add several augmentations to the data provider. If you want to reproduce our paper, just use dataset.py and rife.py.

**Comment by tqyunwuxin** [comment]
> * Many optical flow network provider smaller output than input, if you generate smaller or bigger label, use bilinear resize and rescale value. (such as 2X bilinear resize and *2 all flow value)
> * In my experiments, using census loss or reconstrction loss to finetune optical flow network on Vimeo90K for 5 epoch will slightly improve the final PSNR.
> * Yes, and we add several augmentations to the data provider. If you want to reproduce our paper, just use dataset.py and rife.py.
hi,  the "finetune optical flow network" you mentioned in the second point, do you mean freeze the parameter of the fusionnet  and contextnet?

**Comment by tqyunwuxin** [comment]
hi, the "finetune optical flow network" you mentioned in the second point, do you mean freeze the parameter of the fusionnet and contextnet?


> * Many optical flow network provider smaller output than input, if you generate smaller or bigger label, use bilinear resize and rescale value. (such as 2X bilinear resize and *2 all flow value)
> * In my experiments, using census loss or reconstrction loss to finetune optical flow network on Vimeo90K for 5 epoch will slightly improve the final PSNR.
> * Yes, and we add several augmentations to the data provider. If you want to reproduce our paper, just use dataset.py and rife.py.

**Comment by hzwer** [comment]
https://github.com/hzwer/arXiv2020-RIFE/issues/99#issuecomment-798231465
